### PR TITLE
feat(third_party): add support for `wrap-code` custom class

### PR DESCRIPTION
**Pull request by dandhlee**
Adding support for `wrap-code` class on function/method signatures to allow them to be shown over multiple lines via template and cloud site work, without having the language owners do this work on their own.

I've carefully added the class only towards the signatures and for no other code blocks. This should only affect the signatures. The changed behavior should only be visible if the syntax is too large to fit in the screen and will show over multiple lines. For existing multi-line code blocks, I've confirmed that there is no change in visual layout with the new style applied given that each line does not create the horizontal scroll bar. For one line code blocks, the visual layout also does not change.

This PR by itself will not be adding the desired support but will be the first part to ensure that the class exists for the cloud site to be able to pick up and apply the new style. For the implementation, see cl/407205914.

Currently only enabled for Python.

Towards b/199327840.

- [x] Tests pass

### CODE REVIEW

**Review by tbpg — CHANGES_REQUESTED** [review]
After reviewing with the team, we might want to restrict this to just Python for now.

**Comment by dandhlee** [comment]
* After a review from the devsite team, we've decided to move forward with `wrap-code` that is more of a descriptive name than `prettywrap`.
* Changed to have this only enabled for Python for now, other languages are welcome to join.
* Updated description above to reflect both changes.

Please take a look again!

File: third_party/docfx/templates/devsite/UniversalReference.common.js [hunk 1]
```diff
@@ -109,6 +109,10 @@ function handleItem(vm, gitContribute, gitUrlPattern) {
   if (vm.inheritance) {
     normalizeLanguageValuePairs(vm.inheritance).forEach(handleInheritance);
   }
+
+  if (vm.langs && vm.langs[0] === "python") {
```
Comment (tbpg) [inline third_party/docfx/templates/devsite/UniversalReference.common.js:113]: Please add a comment above this line with a little explanation.

Comment (tbpg) [reply third_party/docfx/templates/devsite/UniversalReference.common.js:113]: Also, is `vm.wrapcode` correct? Is all code wrapped, or just certain spots? `vm.wrapdeclarationcode` (:dizzy_face:)? Might not matter.

**Review by tbpg — APPROVED** [review]

File: third_party/docfx/templates/devsite/UniversalReference.common.js (hunk 1 above)
Comment (dandhlee) [reply third_party/docfx/templates/devsite/UniversalReference.common.js:114]: Let's use `wrapdeclarationcode` :P

Added comment!

**Pull request merged**